.deprecated/deprecated_complete_collection.yml

```
@@ -0,0 +1,75 @@
+!!omap
+- metaData:
+    id: "Ecoli-GEM"
+    name: "The Consensus Genome-Scale Metabolic Model of Escherichia coli"
+    date: "2025-06-16"
+    version: ""
```

How about adding a suggestive name for this file?
```suggestion
    version: "deprecated"
```